site/cds_rdm/views.py

```diff
@@ -30,20 +27,11 @@ def create_blueprint(app):
 
 def index():
     """Frontpage."""
-    anonymous_identity = AnonymousIdentity()
-    anonymous_identity.provides.add(any_user)
     featured_communities_search = current_communities.service.featured_search(
-        anonymous_identity
+        g.identity
```

Comment:
I have added the search as anonymous user as it was discussed with Nico due concerns about caching in the future, here the comment: https://github.com/CERNDocumentServer/cds-rdm/pull/313#discussion_r1910651280